drivers/ufs/ufs.c

```diff
@@ -511,6 +513,8 @@ static void ufs_query(uint8_t op, uint8_t idn, uint8_t index, uint8_t sel,
 	case QUERY_WRITE_ATTR:
 		assert(((buf & 3) == 0) && (size != 0));
 		break;
+	default:
+		break;
```

Comment:
As with the others, I think good to add a comment indicating that this expected to be empty.